src/entry_points/eest_utils.py

```diff
@@ -0,0 +1,133 @@
+"""
+CLI interface for common EEST shortcuts and utilities.
+"""
+
+import logging
+import os
+import shutil
+import subprocess
+import sys
+
+import click
+import colorlog
+
+
+def setup_logger():
+    """
+    Custom logger setup for EEST CLI.
+    """
+    handler = colorlog.StreamHandler()
+    handler.setFormatter(
+        colorlog.ColoredFormatter(
+            "%(log_color)s%(levelname)s%(reset)s[%(asctime)s] %(message)s",
+            datefmt="%m-%d|%H:%M:%S",
+            log_colors={"INFO": "green", "WARNING": "yellow"},
+        )
+    )
+    logger = colorlog.getLogger(__name__)
+    logger.addHandler(handler)
+    logger.setLevel(logging.INFO)
+    return logger
+
+
+logger = setup_logger()
+
+
+@click.group()
+def eest():
+    """
+    EEST commands group for common shortcuts and utilities.
+    """
+    pass
+
+
+@eest.command()
+def init():
+    """
+    Initializes and installs all required packages add dependencies for the repository:
+    ```
+    python3 -m venv ./venv/
+    pip install --upgrade pip
+    pip install -e '.[docs,lint,test]'
+    solc-select install 0.8.22
+    solc-select use latest --always-install
+    ```
+    """
+    venv_create = [sys.executable, "-m", "venv", "./venv/"]
+    logger.info(f"Creating a virtual environment: `{' '.join(venv_create)}`")
+    subprocess.run(venv_create, check=True)
+
+    pip_path = os.path.join(".", "venv", "bin", "pip")
+
+    pip_upgrade = [pip_path, "install", "--upgrade", "pip"]
+    logger.info(f"Upgrading pip to the latest version: `{' '.join(pip_upgrade)}`")
+    subprocess.run(pip_upgrade, check=True)
+
+    pip_install = [pip_path, "install", "-e", ".[docs,lint,test]"]
+    logger.info(f"Installing required packages: `{' '.join(pip_install)}`")
+    subprocess.run(pip_install, check=True)
+
+    solc_select_path = os.path.join(".", "venv", "bin", "solc-select")
+    # solc-select uses $HOME to store the solc binaries, so temporarily change it to the venv
+    original_home = os.environ.get("HOME")
+    os.environ["HOME"] = "./venv"
+
+    # Required for forks <= Constantinople, see important note:
+    # https://github.com/ethereum/solidity/releases/tag/v0.8.22
+    solc_install_0_8_22 = [solc_select_path, "install", "0.8.22"]
```

Comment:
```suggestion
    SOLC_PINNED_VERSION = "0.8.22"
    solc_install_pinned = [solc_select_path, "install", SOLC_PINNED_VERSION]
```
Or something along those lines.